setup.py

```diff
@@ -60,9 +60,9 @@
             'data/busco_db/busco_downloads/lineages/lineage_1/hmms/*',
             'data/busco_db/busco_downloads/lineages/lineage_1/info/*',
             'data/busco_db/busco_downloads/lineages/lineage_1/prfl/*'
-            'data/busco_results/sample1/*',
-            'data/busco_results/sample2/*',
-            'data/busco_results/collated/*',
+            'data/busco_results/sample1/busco_results.tsv',
+            'data/busco_results/sample2/busco_results.tsv',
+            'data/busco_results/collated/busco_results.tsv',
```

Comment:
Hmmm, I'm wondering whether it's also necessary to add here `data/busco_results/*`?

Reply:
Thats what i tried in the first commit:
https://github.com/bokulich-lab/q2-moshpit/pull/175/commits/9c5c47a515bd50b33b71044b70fb3822bb469d7b
but that also did not work, I just tried it explicitly like this to see if this would resolve the problem

Reply:
Sorry, I meant to `add` it here, so that you have both - just a random guess 😅